azure-spring-boot/pom.xml

```diff
@@ -44,6 +44,14 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
```

Comment:
why duplicate here? let's keep dependencies in parents only.

Reply:
In parent it's dependency management, we still need to add it as dependency without specifying version.

Reply:
got it. this is the bug caused the issue. any other newly introduced dependencies missing?

Reply:
This issue happens when user does not user any spring boot starter, by default, spring boot starter will add these dependencies.